components/cam/src/physics/crm/crm_physics.F90

```diff
@@ -813,12 +837,80 @@ subroutine crm_physics_tend(ztodt, state, tend, ptend, pbuf, cam_in, cam_out, &
       if (.not.allocated(ptend%q)) write(*,*) '=== ptend%q not allocated ==='
       if (.not.allocated(ptend%s)) write(*,*) '=== ptend%s not allocated ==='
 
+      ! Load latitude, longitude, and unique column ID for all CRMs
+      allocate(longitude0(ncol))
+      allocate(latitude0 (ncol))
+      allocate(gcolp     (ncol))
+      do icrm = 1 , ncol
+        latitude0 (icrm) = get_rlat_p(lchnk,icrm) * 57.296_r8
+        longitude0(icrm) = get_rlon_p(lchnk,icrm) * 57.296_r8
+        gcolp     (icrm) = get_gcol_p(lchnk,icrm)
+      enddo
+      ! Load the MSA parameters
```

Comment:
I think we need a more descriptive comment here, without the abbreviation. I would suggest something like this:
"set CRM mean state acceleration (MSA) parameters from namelist"

Reply:
Done